packages/editor-ui/src/views/MfaView.vue

```diff
@@ -94,20 +94,20 @@ const onBackClick = () => {
 
 	showRecoveryCodeForm.value = false;
 	hasAnyChanges.value = true;
-	formInputs.value = [mfaTokenFieldWithDefaults()];
+	formInputs.value = [mfaCodeFieldWithDefaults()];
 	emit('onBackClick', MFA_FORM.MFA_RECOVERY_CODE);
 };
 
-const onSubmit = async (form: { token: string; recoveryCode: string }) => {
+const onSubmit = async (form: { mfaCode: string; mfaRecoveryCode: string }) => {
 	formError.value = !showRecoveryCodeForm.value
 		? i18.baseText('mfa.code.invalid')
 		: i18.baseText('mfa.recovery.invalid');
 	emit('submit', form);
 };
 
 const onInput = ({ target: { value, name } }: { target: { value: string; name: string } }) => {
-	const isSubmittingMfaToken = name === 'token';
-	const inputValidLength = isSubmittingMfaToken
+	const isSubmittingMfaCode = name === 'mfaToken';
```

Comment:
Since we are unifying the naming:
```suggestion
	const isSubmittingMfaCode = name === 'mfaCode';
```